pkg/ssh/manager.go

```diff
@@ -48,7 +48,7 @@ func NewForwardManager(ctx context.Context, sshAddr, localInterface, remoteInter
 	}
 }
 
-func (fm *ForwardManager) canAdd(localPort int) error {
+func (fm *ForwardManager) canAdd(localPort int, checkAvailable bool) error {
```

Comment:
`checkAvailable` is not used, doesn't it?

Reply:
I didn't push the last commit 🤦‍♂️